ppr-ui/src/utils/registration-helper.ts

```diff
@@ -379,6 +501,92 @@ export async function setupFinancingStatementDraft (stateModel:StateModelIF, doc
   return stateModel
 }
 
+export function setupStateModelFromAmendmentDraft (stateModel:StateModelIF, draft:DraftIF): StateModelIF {
```

Comment:
Just to clarify, what is this function for vs. setupAmendmentStatementFromDraft  :)

Reply:
setupAmendmentStatementFromDraft makes an API call to get the amendment draft. It then calls setupStateModelFromAmendmentDraft to update the model with the draft data. Splitting into setupStateModelFromAmendmentDraft facilitates unit testing with different scenarios (without making an API call).